### PR TITLE
<Layouter/> - new component

### PR DESCRIPTION
**Pull request by justame**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6368

File: perfer.config.js [hunk 1]
```diff
@@ -256,41 +218,21 @@ const cssFiles = [
   ['Proportion.min.css', 1],
   ['RadioGroup.min.css', 2],
   ['Radio.min.css', 1],
+=======
+>>>>>>> infra - remove scss tokens from unneeded file and update perfer (#6393)
```
Comment (TalZa1810) [inline perfer.config.js]: note that it added some commits from different PRs which are not related to this PR.

**Comment by assafzi** [comment]
UX is here.
Looks good.
2 things to improve.

- [ ] 1. Change component's name to **AnalyticsLayout**
After an internal discussion, Layouter can be confusing (we have a component named layout)

- [ ] Adding minimum width example to the description
**Minimum Width**
Width depends on parent’s container, but cannot be less than 636px.
image

**Comment by assafzi** [comment]
UX Approved.
Thanks.

**Pull request merged**